site/sidebar.ts

```diff
@@ -104,6 +104,10 @@ export const sidebar = [
             text: 'Swap',
             link: '/swap/swap',
           },
+          {
+            text: 'SwapSettingsSlippage',
+            link: '/swap/swap-settings',
```

Comment:
I've been debating calling this page `SwapSettingsSlippage` or `SwapSettings`. 

`SwapSettingsSlippage` is more descriptive and will be easier to find but there is no component called `SwapSettingsSlippage`.